examples/server/README.md

```diff
@@ -276,13 +276,15 @@ Notice that each `probs` is an array of length `n_probs`.
 {
   "assistant_name": "",
   "user_name": "",
-  "default_generation_settings": { ... }
+  "default_generation_settings": { ... },
+  "total_slots": ""
```

Comment:
A nitpick, but it should probably be `"total_slots": 1`, since it's an integer, not a string.

Reply:
updated